pysages/methods/metad.py

```diff
@@ -287,3 +288,24 @@ def sum_of_gaussians(xi, heights, centers, sigmas, periods):
     """
     delta_x = wrap(xi - centers, periods)
     return gaussian(heights, sigmas, delta_x).sum()
+
+
+@dispatch
+def analyze(result: Result[Metadynamics]):
+    """
+    Computes the free energy from the result of a `Metadynamics` run.
+    """
+    method = result.method
+    state = result.states
+
+    P = get_periods(method.cvs)
+    heights = state.heights
+    centers = state.centers
+    sigmas = state.sigmas
+
+    @jit
+    def metapotential(xs):
+        f = vmap(lambda x: sum_of_gaussians(x, heights, centers, sigmas, P))
+        return f(xs)
+
+    return dict(heights=heights, sigmas=sigmas, metapotential=metapotential)
```

Comment:
Maybe that is me, but what exactly is `heights` and how do  I get the free energy profile from there?

Reply:
`heights` in metadynamics refer to the height of the deposited Gaussian potential. It is one of the parameters provided by the user. In standard metadynamics, it is constant that is equal to the user-defined value. For well-tempered metadynamics, it decays exponentially with simulation. It is one of the metric to monitor convergence of the simulations.
To determine free energy, we can use the `metapotential` function. According to standard metadynamics, the free energy is equivalent to the total deposited bias potential. This can be determined using the `metapotential` function. An example is provided in the colab notebook.

Reply:
Thanks for the explanation.
Can we document this for the return of the dictionary with this function?
I think users should be able to find this explanation there.

Reply:
Sure, I can add that. I think we don't need to return `sigmas` here. So, I ll also edit that. Thanks!